src/main/java/se/sundsvall/supportmanagement/integration/db/model/StakeholderEntity.java

```diff
@@ -0,0 +1,235 @@
+        return this;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public StakeholderEntity withLastName(String lastName) {
+        this.lastName = lastName;
+        return this;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public StakeholderEntity withAddress(String address) {
+        this.address = address;
+        return this;
+    }
+
+    public String getCareOf() {
+        return careOf;
+    }
+
+    public void setCareOf(String careOf) {
+        this.careOf = careOf;
+    }
+
+    public StakeholderEntity withCareOf(String careOf) {
+        this.careOf = careOf;
+        return this;
+    }
+
+    public String getZipCode() {
+        return zipCode;
+    }
+
+    public void setZipCode(String zipCode) {
+        this.zipCode = zipCode;
+    }
+
+    public StakeholderEntity withZipCode(String zipCode) {
+        this.zipCode = zipCode;
+        return this;
+    }
+
+    public String getCountry() {
+        return country;
+    }
+
+    public void setCountry(String country) {
+        this.country = country;
+    }
+
+    public StakeholderEntity withCountry(String country) {
+        this.country = country;
+        return this;
+    }
+
+    public List<ContactChannelEntity> getContactChannels() {
+        return contactChannels;
+    }
+
+    public void setContactChannels(List<ContactChannelEntity> contactChannels) {
+        this.contactChannels = contactChannels;
+    }
+
+    public StakeholderEntity withContactChannels(List<ContactChannelEntity> contactChannels) {
+        this.contactChannels = contactChannels;
+        return this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        StakeholderEntity that = (StakeholderEntity) o;
+        return id == that.id && Objects.equals(errandEntity, that.errandEntity) && Objects.equals(stakeholderId, that.stakeholderId) && Objects.equals(type, that.type) && Objects.equals(firstName, that.firstName) && Objects.equals(lastName, that.lastName) && Objects.equals(address, that.address) && Objects.equals(careOf, that.careOf) && Objects.equals(zipCode, that.zipCode) && Objects.equals(country, that.country) && Objects.equals(contactChannels, that.contactChannels);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, errandEntity, stakeholderId, type, firstName, lastName, address, careOf, zipCode, country, contactChannels);
+    }
+
+    @Override
+    public String toString() {
+        final StringBuilder sb = new StringBuilder("StakeholderEntity{");
+        sb.append("id=").append(id);
+        sb.append(", errandEntity=").append(errandEntity);
```

Comment:
This will cause a stack overflow when this or the ErrandEntity object is printed.

Reply:
Fixed